src/components/search-box.vue

```diff
@@ -1,26 +1,27 @@
 <template>
-  <div class="x-input-group x-input-group--card x-border-color--lead">
+  <div class="x-input-group x-input-group-lead x-rounded-sm">
     <SearchInput
```

Comment:
Font size has changed from `16px` to inheriting `14px`, I think we should change it in the home template from `x-archetype-utils` so they're the same.

Reply:
done in this other PR: https://github.com/empathyco/x/pull/1079